go/base/static/js/src/apps/dialogue/states/states.js

```diff
@@ -18,6 +18,8 @@
       ParametricEndpointView = endpoints.ParametricEndpointView,
       AligningEndpointCollection = endpoints.AligningEndpointCollection;
 
+  var maxChars = 140;
```

Comment:
Are we sure we want to use `140` instead of `160`? `140` seems safer, since operator-that-will-not-be-named sometimes restricts USSD session messages to 140 instead of 160, and since other channels like twitter have 140 as a restriction. That said, 160 is quite common for USSD, in SA at least, not too sure on other countries. I just get the feeling most dialogues will have states exceeding 140 chars because of this, and the feature loses its usefulness.